environment/src/main/java/com/sequenceiq/environment/environment/EnvironmentStatus.java

```diff
@@ -197,7 +197,9 @@ public enum EnvironmentStatus {
     );
 
     public static final Set<EnvironmentStatus> VERTICAL_SCALABLE_STATES = Set.of(
-            ENV_STOPPED
+            ENV_STOPPED,
+            VERTICAL_SCALE_ON_FREEIPA_FAILED,
+            VERTICAL_SCALE_ON_FREEIPA_IN_PROGRESS
```

Comment:
@doktoric what is the proper env/freeipa status when we should accepting the vertical scale?